.gitignore

```diff
@@ -20,3 +20,7 @@ wasm/test_page/node_modules
 build-wasm
 models
 wasm/test_page/bergamot-translator-worker.*
+build*
```

Comment:
There's a `build-wasm.sh` which will be gitignore-d after this. Careful when you use regexes?

Reply:
Commit 941de68